backend/src/utils/fullNameCreator.ts

```diff
@@ -0,0 +1 @@
+export const fullNameCreator = (firstName: string, lastName: string) => `${firstName} ${lastName}`;
```

Comment:
pls, make name of the function to sound like a verb. Usually classes are named as nouns.

Reply:
`combineFullName` for example